index.js

```diff
@@ -5,6 +5,26 @@ function priceFormat(amountInCents, currencySymbol) {
   return currencySymbol + (amountInCents / 100).toFixed(2).replace(/\B(?=(\d{3})+(?!\d))/g, ',');
 }
 
+function priceFormatMultiCurrency(unitAmount, currencyCode, currencySymbol) {
+  currencyCode = currencyCode || 'usd';
+  currencySymbol = currencySymbol || '$';
+  const intlFormat = new Intl.NumberFormat(undefined, {
+    currency: currencyCode,
+    style: 'currency'
+  });
+
+  const { minimumFractionDigits: currencyDecimalPlaces } = intlFormat.resolvedOptions();
+
+  const formattedAmount = unitAmount / (10 ** currencyDecimalPlaces);
+
+  //add comma and decimal separators
+  const withSeparatorsAmount = intlFormat.format(formattedAmount);
+  //remove leading prefix
+  const withoutPrefixAmount = withSeparatorsAmount.replace(/^[^\d]*/g, '');
```

Comment:
I’m a little confused about this string manipulation. I think the output of `new Intl.NumberFormat(undefined, { currency: currencyCode, style: 'currency' }).format(unitAmount)` should be used directly. We don’t want to be in the business of recreating what the library already does – certain currencies are formatted with the currency symbol after the amount, for example.

Reply:
Good call! Originally I had these extra steps in to keep the prefix portion matching with its entry in `currency_symbols` (e.g. `cad` would result in "$3.00" versus "CA$3.00"). Though with these only being used for display purposes through `totalLineOneMulticurrency` and `totalLineTwoMulticurrency` there shouldn't be a need to keep these in sync over established formatting. Refactoring to return result of `intlFormat.format(formattedAmount)`.